tests.js

```diff
@@ -166,6 +174,38 @@ describe('Soft Delete plugin tests', () => {
           });
       });
     });
+
+    describe('when deletedValue and nonDeletedValue are overridden', () => {
+      before(() => {
+        return knex.schema.table('TestObjects', (table) => {
```

Comment:
I tried to
```js
before(createDeletedAtColumn(knex))
after(removeDeletedAtColumn(knex))
```
And then, at the beggining of the file, outside the first `describe` scope, declare `createDeletedAtColumn` and `removeDeletedAtColumn` but it didn't work. 
It says that knex is undefined. javascript shenanigans I'd say. Anyway, open to suggestions